zookeeper/tf/dataset.py

```diff
@@ -5,7 +5,7 @@
 import tensorflow_datasets as tfds
 
 from zookeeper.core import utils
-from zookeeper.core.field import Field
+from zookeeper.core.field import Field  # type: ignore
```

Comment:
Do you think it makes sense to make this ignore a bit stricter `# pytype: disable=<error-type>`?

Reply:
Yeah you're right it's not ideal.

The current method disables any typechecking that relies on the type of what's being imported, in this case `Field` - see the [docs here](https://google.github.io/pytype/faq.html#how-do-i-disable-all-pytype-checks-for-a-particular-import).

If I add `# pytype: disable=annotation-type-mismatch` on the same line as the import, replacing the current comment, it doesn't have any effect, and pytype errors are still thrown throughout the file. If I instead add `# pytype: disable=annotation-type-mismatch` as a standalone line at the top of the file, it disables this check for everything, not just things involving `Field` (so `a: int = 5.4` would not be caught).

I'm not sure either option is obviously better than the other, what do you think?

Reply:
Makes sense, let's keep it that way.